src/_P092_DLbus.ino

```diff
@@ -0,0 +1,1288 @@
+  P092_DataSettings.IdxVolume               = P092_DataSettings.IdxHmRegister + 1;
+  P092_DataSettings.IdxHeatMeter1           = P092_DataSettings.IdxVolume + P092_DataSettings.VolumeBytes;
+  P092_DataSettings.IdxkWh1                 = P092_DataSettings.IdxHeatMeter1 + P092_DataSettings.CurrentHmBytes;
+  P092_DataSettings.IdxMWh1                 = P092_DataSettings.IdxkWh1 + 2;
+  P092_DataSettings.IdxHeatMeter2           = P092_DataSettings.IdxMWh1 + P092_DataSettings.MWhBytes;
+  P092_DataSettings.IdxkWh2                 = P092_DataSettings.IdxHeatMeter2 + P092_DataSettings.CurrentHmBytes;
+  P092_DataSettings.IdxMWh2                 = P092_DataSettings.IdxkWh2 + 2;
+  P092_DataSettings.IdxHeatMeter3           = P092_DataSettings.IdxMWh2 + P092_DataSettings.MWhBytes;
+  P092_DataSettings.IdxkWh3                 = P092_DataSettings.IdxHeatMeter3 + P092_DataSettings.CurrentHmBytes;
+  P092_DataSettings.IdxMWh3                 = P092_DataSettings.IdxkWh3 + 2;
+}
+/*********************************************************************************************\
+  DLBus subs to get values from the P092_receiving bitstream
+\*********************************************************************************************/
+// one data frame has <P092_DataSettings.DataBytes> data bytes + SYNC, e.g. 64 * (8+1+1) + 16 = 656
+// 656 * 2 = 1312 (twice as much as a data frame is saved
+// so there's one complete data frame
+
+#define DLbus_MaxDataBytes 64
+#define DLbus_AdditionalRecBytes 2
+#define DLbus_StopBits 1
+#define DLbus_StartBits 1
+#define DLBus_SyncBits 16
+#define DLBus_ReserveBytes 20
+#define DLBus_BitChangeFactor 2
+#define DLbus_MaxDataBits (((DLbus_MaxDataBytes + DLbus_AdditionalRecBytes) * (DLbus_StartBits + 8 + DLbus_StopBits) + DLBus_SyncBits) * DLBus_BitChangeFactor) + DLBus_ReserveBytes
+        // MaxDataBits is double of the maximum bit length because each bit change is stored
+        // (((64+2) * (8+1+1) + 16) * 2) + 50 = 1402 bytes
+
+// variables used in interrupt function
+volatile static uint8_t* DLbus_PtrChangeBitStream;          // pointer to received bit change stream
+volatile uint16_t DLbus_pulse_count;                        // number of received pulses
+volatile uint16_t DLbus_pulse_number;                       // max naumber of the received pulses
+volatile uint32_t DLbus_TimeLastBitChange = 0;     			    // remember time of last transition
+volatile uint16_t DLbus_MinPulseWidth, DLbus_MaxPulseWidth, DLbus_MinDoublePulseWidth, DLbus_MaxDoublePulseWidth;
+
+static uint8_t DLbus_ChangeBitStream[DLbus_MaxDataBits];    // received bit change stream (each bit change is extended to uint8_t, containing the timing flags)
+uint8_t DLbus_ByteStream[DLbus_MaxDataBits / 8 + 1];  // every bit gets sorted into a bitmap
+uint16_t DLbus_bit_number;                                  // bit number of the received DLbus_ChangeBitStream
+
+uint8_t DLbus_WrongTimeCnt;
+uint16_t DLbus_WrongTimings[5][6];
```

Comment:
And another 60 bytes allocated for `DLbus_WrongTimings` on the stack.  (sys stack, not cont stack)
This is allocated regardless whether the plugin is being used.
The same applies for `DLbus_ByteStream`, which is now already close to 200 bytes.

Reply:
Changed in the next version of _P092_DLbus.ino